firestore/swift/firestore-smoketest/SolutionArraysViewController.swift

```diff
@@ -21,94 +21,94 @@ import FirebaseFirestore
 
 class SolutionArraysViewController: UIViewController {
 
-    var db: Firestore!
-
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        db = Firestore.firestore()
+  var db: Firestore!
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    db = Firestore.firestore()
+  }
+
+  func queryInCategory() async {
+    // [START query_in_category]
+    do {
+      let snapshot = try await db.collection("posts")
+        .whereField("categories.cats", isEqualTo: true)
+        .getDocuments()
+    } catch {
+      print("Error: \(error)")
     }
-
-    func queryInCategory() {
-        // [START query_in_category]
-        db.collection("posts")
-            .whereField("categories.cats", isEqualTo: true)
-            .getDocuments() { (querySnapshot, err) in
-
-                // ...
-
-        }
-        // [END query_in_category]
+    // [END query_in_category]
+  }
+
+  func queryInCategoryTimestamp() {
+    // [START query_in_category_timestamp_invalid]
+    db.collection("posts")
+      .whereField("categories.cats", isEqualTo: true)
+      .order(by: "timestamp")
+    // [END query_in_category_timestamp_invalid]
+
+    // [START query_in_category_timestamp]
+    db.collection("posts")
+      .whereField("categories.cats", isGreaterThan: 0)
+      .order(by: "categories.cats")
+    // [END query_in_category_timestamp]
+  }
+
+  // [START post_with_array]
+  struct PostArray {
+
+    let title: String
+    let categories: [String]
+
+    init(title: String, categories: [String]) {
```

Comment:
Same - aren't these synthesised by the compiler?

Reply:
Done.